ledger/ledger-api-common/src/main/scala/com/digitalasset/platform/akkastreams/dispatcher/SignalDispatcher.scala

```diff
@@ -63,14 +63,17 @@ class SignalDispatcher private () extends AutoCloseable {
   /** Closes this SignalDispatcher.
     * For any downstream with pending signals, at least one such signal will be sent first.
     */
-  def close(): Unit =
+  def close(): Future[Unit] = {
```

Comment:
How about release or shutdown in order to differentiate to the common conception of `close` being synchronous?